tests/Rules/Fixture/laravel-mailable-method.php

```diff
@@ -17,4 +17,14 @@ public function build()
             'foo' => 'bar',
         ]);
     }
+
+    /**
+     * @return $this
+     */
+    public function buildWith()
+    {
+        return $this->view('foo')->with([
+            'foo' => 'bar',
+        ]);
+    }
```

Comment:
```suggestion
    }

    /**
     * Get the message content definition.
     */
    public function content(): Content
    {
        return new Content(
            view: 'foo',
            with: [
                'foo' => 'bar',
            ],
        );
    }
```

I am trying to make bladestan work for this one, but not much hope as this is my first phpstan contribution 🤞🏻